qa/L0_batcher/batcher_test.py

```diff
@@ -35,8 +35,15 @@
 import numpy as np
 import infer_util as iu
 import test_util as tu
+from functools import partial
 
 import tritonclient.grpc as grpcclient
+from tritonclient.utils import *
```

Comment:
I believe `import *` is not a good practice as it may pollute the namespace and override functions (although not likely to happen in this case)